Descriptio: ticket layout fixes
 Avoid printing ticket number multiple times
From: Krzysztof Baranski <kbaransk@gmail.com>
Last-Update: 14-10-05

--- a/printing/print-cups.cpp
+++ b/printing/print-cups.cpp
@@ -510,7 +510,7 @@
   fm = painter.fontMetrics();
   text = ptInfo.thTicket;
   textWidth = fm.size(Qt::TextExpandTabs | Qt::TextDontClip, text);
-  painter.drawText(printer.width() - textWidth.width() - Margin, yPos, text);
+  //painter.drawText(printer.width() - textWidth.width() - Margin, yPos, text);
   yPos = yPos + fm.lineSpacing();
 
 
@@ -963,7 +963,7 @@
     code.remove(QChar(','), Qt::CaseSensitive);
     code.remove(QChar('.'), Qt::CaseSensitive);
     code = "*"+code+"*"; //for code 39 to work.
-    painter.drawText((printer.width()/2)-(fm.size(Qt::TextExpandTabs | Qt::TextDontClip, code).width()/2)-Margin, Margin+yPos, code);
+    //painter.drawText((printer.width()/2)-(fm.size(Qt::TextExpandTabs | Qt::TextDontClip, code).width()/2)-Margin, Margin+yPos, code);
     //now draw the number in text (no barcode) below.
     tmpFont = QFont("Bitstream Vera Sans", textSize);
     painter.setFont(tmpFont);
